crowbar_framework/app/models/node_object.rb

```diff
@@ -1449,7 +1449,7 @@ def disk_release(device, owner)
     if device
       crowbar_wall[:claimed_disks] ||= {}
 
-      unless disk_owner(device) == owner
+      if owner.empty? || disk_owner(device) != owner
```

Comment:
I'm fine with that change, but can you also do it in chef/cookbooks/barclamp/libraries/barclamp_library.rb since we keep the code in sync here and there.